package.json

```
@@ -13,14 +13,32 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-scripts": "5.0.0",
-    "web-vitals": "2.1.4"
+    "web-vitals": "2.1.4",
+    "express": "^4.19.2",
+    "cors": "^2.8.5",
+    "dotenv": "^16.4.5",
+    "axios": "^1.6.8",
```

Biome (2.1.2)

[error] The key axios was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)